pandas/core/base.py

```diff
@@ -1182,6 +1182,16 @@ def factorize(self, sort: bool = False, na_sentinel: Optional[int] = -1):
         >>> ser.searchsorted([1, 3], side='right')
         array([1, 3])
 
+        >>> ser = pd.Series(pd.to_datetime(['21.10.2020', '22.10.2020', '23.10.2020']))
+        >>> ser
+        0   2020-10-21
+        1   2020-10-22
+        2   2020-10-23
+        dtype: datetime64[ns]
+
+        >>> ser.searchsorted('3/14/2000')
```

Comment:
sorry if i'm being slow but shouldn't this return 0?

Reply:
You are absolutely right. thanks

Reply:
also need to use the same dates in the constructed series and the output 
(that's why CI is red atm. Once these are fixed I think you're ready to go)